transforms/code/header_cleanser/python/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM docker.io/python:3.11.9-slim-bullseye
+
+RUN pip install --upgrade pip 
+
+# Install system dependencies, including libgomp1
+RUN apt-get update && apt-get install -y \
+    libgomp1 \
+    && rm -rf /var/lib/apt/lists/*
+    
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+# Create a user and use it to run the transform
+RUN useradd -ms /bin/bash dpk
+USER dpk
+WORKDIR /home/dpk
+
```

Comment:
Per new PR and feature, can you add

`RUN mkdir input && mkdir output`

This allows mounting of local file system for processing data with the image.  See pr #394.

Reply:
Addressed in the latest commit